src/app/containers/Include/amp/Idt2Amp.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import AmpIncludeContainer from './Idt2Amp';
```

Comment:
```suggestion
import Idt2Amp from './Idt2Amp';
```
Just renaming the export to match the default export of `Idt2Amp.jsx`